website/index.html

```diff
@@ -230,7 +230,10 @@
               Find useful forks
             </button>
           </p>
-        </div>
+          </div>
```

Comment:
The alignment of those `div`s is wrong.
Plus, as discussed, the filter field shouldn't appear when there isn't even a repository provided yet.

Reply:
> The alignment of those divs is wrong.

After formatting, you can see that in fact the alignment of those elements is wrong originally.

> Plus, as discussed, the filter field shouldn't appear when there isn't even a repository provided yet.

You will be satisfied after new commits.